### PR TITLE
Add support for LinkedDataSignature2020.

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

File: schemas/linkedDataSignature2020.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+'use strict';
+
+const w3cDateTime = require('./w3cDateTime');
+const identifier = require('./identifier');
+
+const baseSignature = {
+  title: 'Linked Data Signature',
+  description: 'A Linked Data digital signature.',
+  // NOTE: id is not required
+  required: ['type', 'created', 'proofValue'],
```
Comment (dlongley) [inline schemas/linkedDataSignature2020.js:13]: Why isn't `verificationMethod` required?

File: schemas/linkedDataSignature2020.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  type: 'object',
+  properties: {
+    id: identifier(),
+    type: {
+      title: 'Linked Data Signature Type',
+      type: 'string',
+      enum: ['Ed25519Signature2020']
+    },
+    creator: identifier(),
```
Comment (dlongley) [inline schemas/linkedDataSignature2020.js:22]:
```suggestion
```

File: schemas/linkedDataSignature2020.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      anyOf: [{
+        required: ['creator'],
+        not: {required: ['verificationMethod']},
+      }, {
+        required: ['verificationMethod'],
+        not: {required: ['creator']},
+      }]
+    }
+  ]
+};
```
Comment (dlongley) [inline schemas/linkedDataSignature2020.js:47]: This should be simplified to just use `verificationMethod`, `creator` is obsolete.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: schemas/linkedDataSignature2020.js (hunk 3 above)
Comment (msporny) [reply schemas/linkedDataSignature2020.js]: Fixed in be0dcee8b742ff4328dc606c3215d5c01d85177e.

File: schemas/linkedDataSignature2020.js (hunk 1 above)
Comment (msporny) [reply schemas/linkedDataSignature2020.js]: Fixed in be0dcee8b742ff4328dc606c3215d5c01d85177e.

File: schemas/linkedDataSignature2020.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2012-2020 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline schemas/linkedDataSignature2020.js:2]:
```suggestion
 * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
```

File: schemas/linkedDataSignature2020.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ */
+'use strict';
+
+const w3cDateTime = require('./w3cDateTime');
+const identifier = require('./identifier');
+
+const signature = {
+  title: 'An Ed25519 Linked Data Signature',
```
Comment (dlongley) [inline schemas/linkedDataSignature2020.js:10]: This should either be something like "2020 Linked Data Signatures" and allow for more than one suite, or it should just be named `Ed25519Signature2020`, right?

**Review by dlongley — APPROVED** [review]
This seems ok, but see my comments on naming. Thanks!

File: schemas/linkedDataSignature2020.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const w3cDateTime = require('./w3cDateTime');
+const identifier = require('./identifier');
+
+const signature = {
+  title: 'An Ed25519 Linked Data Signature',
```
Comment (msporny) [inline schemas/linkedDataSignature2020.js:10]:
```suggestion
  title: 'Ed25519Signature2020',
```

File: schemas/linkedDataSignature2020.js (hunk 5 above)
Comment (msporny) [reply schemas/linkedDataSignature2020.js]: Renamed to Ed25519Signature2020.

File: schemas/linkedDataSignature2020.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+      description: 'The multibase base58-btc encoding of the result of ' +
+        'the signature algorithm.',
+      type: 'string'
+    },
+    verificationMethod: identifier(),
+  },
+};
+
+const schema = {
+  title: 'Ed25519 Linked Data Signatures',
```
Comment (msporny) [inline schemas/linkedDataSignature2020.js:34]:
```suggestion
  title: 'One or more Ed25519 signatures (2020 cryptosuite)',
```

**Pull request merged**